src/meta/meta_turtle.c

```diff
@@ -527,6 +527,17 @@ __wt_turtle_init(WT_SESSION_IMPL *session, bool verify_meta, const char *cfg[])
         }
 
         if (ret != 0) {
+            if (F_ISSET(conn, WT_CONN_LIVE_RESTORE_FS))
```

Comment:
Shouldn't the fact that conn_salvage is set also mean that we error out? This seems to allow for a successful salvage of the turtle file then running live restore on top of it.

Reply:
Is that an issue? I figure the only concern is if salvage overwrites the live restore state by deleting and replacing the turtle file. If we don't delete it then we can read the live restore state (aborting if it's invalid) from the turtle and retain the correct LR state.

Reply:
I can't imagine a scenario where we'd both want to salvage and continue a live restore. To arrive there something else has gone very wrong. I would say if we detect or are in a salvage scenario and have a live restore file system we should error out. Trying to have the two work together is overkill.

Reply:
SGTM. I've reworked the change to ban live restore and salvage from being enabled at the same time.